cmd/pg/catchup_fetch.go

```diff
@@ -18,10 +20,12 @@ var catchupFetchCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		folder, err := internal.ConfigureFolder()
 		tracelog.ErrorLogger.FatalOnError(err)
-		internal.HandleCatchupFetch(folder, args[0], args[1])
+		internal.HandleCatchupFetch(folder, args[0], args[1], useNewUnwrap)
 	},
 }
 
 func init() {
+	catchupFetchCmd.Flags().BoolVar(&useNewUnwrap, "use-new-unwrap",
```

Comment:
-> reverse-unpack
-> use-new-unwrap
Use common flag.
For example:
- someone used "catchup --use-new-unwrap" and get an error due to a bug
- After that, this person don't understand that it would be better to stop using  "backup-fetch --reverse-unpack" due to avoiding potential errors.